scripts/scp_to_manifest.py

```diff
@@ -17,34 +17,79 @@
 import os
 
 import librosa as l
+from sklearn.model_selection import StratifiedShuffleSplit
 from tqdm import tqdm
 
 
-def main(scp, id, out):
+"""
+This scipt take scp file which is in format 
+<absolute  filename> 
+to manifest json file. 
+Args: 
+--scp: scp file name
+--id: index of speaker label in filename present in scp file that is separated by '/'
+--out: output manifest file name
+--split: True / False if you would want to split the  manifest file for training purposes
+        you may not need this for test set. output file names is <out>_<train/dev>.json
+        Defaults to False
+"""
+
+
+def write_file(name, lines, idx):
+    with open(name, 'w') as fout:
+        for i in idx:
+            dic = lines[i]
+            json.dump(dic, fout)
+            fout.write('\n')
+    print("wrote", name)
```

Comment:
Can you use the logging library. Not nemo.logging just import logging (from python's stdlib)

Reply:
yes,